src/os/bluestore/BlueStore.cc

```diff
@@ -2003,6 +2005,28 @@ void BlueStore::ExtentMap::update(KeyValueDB::Transaction t,
   }
 }
 
+bid_t BlueStore::ExtentMap::allocate_spanning_blob_id()
```

Comment:
> The return variable is a right value, should we make it const?

I guess what does kefu mean is to make the method itself const, not the return value...